src/main/java/seedu/address/model/profile/Profile.java

```diff
@@ -223,7 +223,11 @@ public int hashCode() {
 
     @Override
     public int compareTo(Profile other) {
```

Comment:
Liked that you took into account the orderings for the profiles with the same name. 👍